Userland/Libraries/LibGfx/Painter.cpp

```diff
@@ -64,7 +64,6 @@ Painter::Painter(Gfx::Bitmap& bitmap)
     VERIFY(bitmap.physical_width() % scale == 0);
     VERIFY(bitmap.physical_height() % scale == 0);
     m_state_stack.append(State());
-    state().font = &FontDatabase::default_font();
```

Comment:
`font` needs to be null initialized after these changes.

`state().font = nullptr;`

Or add a default initializer for font to the `State` struct.